caddyhttp/httpserver/pathcleaner.go

```diff
@@ -0,0 +1,75 @@
+package httpserver
+
+import (
+	"math/rand"
+	"path"
+	"strings"
+	"time"
+)
+
+// A proxy function to prevent one or more of the path cleanup operations:
```

Comment:
Now you know how "limited" my knowledge is when it comes to golang. I never read the godoc conventions before receiving this comment. ;-)